recipes/gempy/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "gempy" %}
+{% set version = "2.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 237ba93a0eec5b41068e94977a10aae285aa46f1a527ae2280817975239e6a25
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pandas
+    - matplotlib
+    - numpy
+    - networkx
+    - pyvista
+    - scikit-image >=0.17
+    - seaborn >=0.9
+    - Theano >=1.0.4
```

Comment:
Conda packages should be lower-case, so please replace this with `theano`.

Reply:
Done